CHANGELOG.md

```diff
@@ -7120,6 +7119,8 @@ Swift 1.0
 [SE-0210]: <https://github.com/apple/swift-evolution/blob/master/proposals/0210-key-path-offset.md>
 [SE-0211]: <https://github.com/apple/swift-evolution/blob/master/proposals/0211-unicode-scalar-properties.md>
 [SE-0212]: <https://github.com/apple/swift-evolution/blob/master/proposals/0212-compiler-version-directive.md>
+[SE-0213]: <https://github.com/apple/swift-evolution/blob/master/proposals/0213-literal-init-via-coercion.md>
```

Comment:
Do we need these links in this PR?

Reply:
No, not in this PR. Shall I remove them?

(I haven't got commit access, but I wanted to keep the proposals list up-to-date).

Should I also undo the re-wrapping (to 80 columns) of your changelog entry?